lib/src/db.rs

```diff
@@ -467,21 +471,26 @@ impl Storelike for Db {
 
     #[instrument(skip(self))]
     fn all_resources(&self, include_external: bool) -> ResourceCollection {
-        let mut resources: ResourceCollection = Vec::new();
-        let self_url = self
-            .get_self_url()
-            .expect("No self URL set, is required in DB");
-        for item in self.resources.into_iter() {
+        // FIXME: Try rossbeam channels here
```

Comment:
What are rossbeam channels? Maybe create an issue for this and mention it here?

Reply:
This is my todo.